bears/vcs/git/GitCommitBear.py

```diff
@@ -24,4 +25,33 @@ def get_remotes():
 
     def get_head_commit(self):
         with change_directory(self.get_config_dir() or os.getcwd()):
+            command = self.github_pull_request_temporary_merge_commit()
+            if command:
+                return run_shell_command(command)
+
             return run_shell_command('git log -1 --pretty=%B')
+
+    def github_pull_request_temporary_merge_commit(self):
```

Comment:
I think we might want to change this function name into something that explains this function's functionality.